packages/gatsby/src/utils/flags.ts

```diff
@@ -29,14 +28,16 @@ export const satisfiesSemvers = (
   return result
 }
 
-export type fitnessEnum = true | false | "OPT_IN"
+export type fitnessEnum = true | false | "OPT_IN" | "LOCKED_IN"
+
+export const DO_NOT_TRACK_THIS_FLAG = Symbol(`DO_NOT_TRACK_THIS_FLAG`)
 
 export interface IFlag {
   name: string
   env: string
   description: string
   command: executingCommand
-  telemetryId: string
+  telemetryId: string | typeof DO_NOT_TRACK_THIS_FLAG
```

Comment:
Why we don't make it optional instead?

```suggestion
  telemetryId?: string | undefined
```

Reply:
I wanted this to be "hard" not to track feature usage. Making this field optional will make it easy when adding new feature to skip adding telemetry id. Using explicit (symbol) makes it very intentional which features are not supposed to be tracked (anymore)

Reply:
So situation of "opting out" of tracking instead of "opting in" ;)

Reply:
Alternatively I could do `string | false` instead of symbol to achieve same thing

Reply:
just not `undefined` / optional

Reply:
Do you think `""` (empty string) will be too easy? So `telemetryId: string` but skip if it's empty string? We should probably skip tracking in this case anyway.

Reply:
But I don't insist here. Feel free to ignore this nit. Making it explicit makes perfect sense. Just looks a bit weird :)

Reply:
I think I'll go with `false`. In practice empty string will work the same technically - I just like `false` over empty string as it has some semantic differince between texts (tho this is really NITy territory ;p). As long as premise of using something explicit to opt out instead of optional opt-in is fine - I'm happy with whatever ;)

Reply:
https://github.com/gatsbyjs/gatsby/pull/28787/commits/e22fc5e9a42e898fb509d6227a61c410d86cfb78 is switch to `false` instead of weird Symbol ;) (I will get to use Symbol at some point!)